cmd/paths.go

```diff
@@ -1,17 +1,17 @@
 package cmd
 
 import (
+	"context"
 	"encoding/json"
 	"fmt"
 	"io"
 	"io/ioutil"
-	"os"
-	"path"
 	"path/filepath"
+	"sort"
 	"strings"
 
 	"github.com/cosmos/relayer/v2/relayer"
-	"github.com/go-git/go-git/v5"
+	"github.com/google/go-github/github"
```

Comment:
This should be `github.com/google/go-github/v43/github` to match the version of the module we are using elsewhere in the project.

As a side note, it looks like the current release is v45, but updating to that is outside the scope of this PR.